config/config.php

```diff
@@ -2,203 +2,285 @@
-    // to GraphQL::schema() or when the route is used without the graphql_schema
-    // parameter.
+    /*
+    |--------------------------------------------------------------------------
+    | The name of the default schema used when no argument is provided
+    | to GraphQL::schema() or when the route is used without the
+    | graphql_schema parameter
+    |--------------------------------------------------------------------------
+    |
+    */
+
     'default_schema' => 'default',
 
-    // The schemas for query and/or mutation. It expects an array of schemas to provide
-    // both the 'query' fields and the 'mutation' fields.
-    //
-    // You can also provide a middleware that will only apply to the given schema
-    //
-    // Example:
-    //
-    //  'schema' => 'default',
-    //
-    //  'schemas' => [
-    //      'default' => [
-    //          'query' => [
-    //              'users' => 'App\GraphQL\Query\UsersQuery'
-    //          ],
-    //          'mutation' => [
-    //
-    //          ]
-    //      ],
-    //      'user' => [
-    //          'query' => [
-    //              'profile' => 'App\GraphQL\Query\ProfileQuery'
-    //          ],
-    //          'mutation' => [
-    //
-    //          ],
-    //          'middleware' => ['auth'],
-    //      ],
-    //      'user/me' => [
-    //          'query' => [
-    //              'profile' => 'App\GraphQL\Query\MyProfileQuery'
-    //          ],
-    //          'mutation' => [
-    //
-    //          ],
-    //          'middleware' => ['auth'],
-    //      ],
-    //  ]
-    //
+    /*
+    |--------------------------------------------------------------------------
+    | The schemas for query and/or mutation. It expects an array of schemas
+    | to provide both the 'query' fields and the 'mutation' fields. You can
+    | also provide a middleware that will only apply to the given schema
+    |--------------------------------------------------------------------------
+    |
+    | Example:
+    |
+    |  'schema' => 'default',
+    |
+    |  'schemas' => [
+    |      'default' => [
+    |          'query' => [
+    |              'users' => 'App\GraphQL\Queries\UsersQuery'
+    |          ],
+    |          'mutation' => [
+    |
+    |          ]
+    |      ],
+    |      'user' => [
+    |          'query' => [
+    |              'profile' => 'App\GraphQL\Queries\ProfileQuery'
+    |          ],
+    |          'mutation' => [
+    |
+    |          ],
+    |          'middleware' => ['auth'],
+    |      ],
+    |      'user/me' => [
+    |          'query' => [
+    |              'profile' => 'App\GraphQL\Queries\MyProfileQuery'
+    |          ],
+    |          'mutation' => [
+    |
+    |          ],
+    |          'middleware' => ['auth'],
+    |      ],
+    |  ]
+    |
+    */
+
     'schemas' => [
+
         'default' => [
-            'query' => [
-                // 'example_query' => ExampleQuery::class,
```

Comment:
I prefer to keep these; also I think it's the reason the integration tests break